elastio-s3-changelog/README.md

```diff
@@ -0,0 +1,38 @@
+# S3 Changelog
+
+This CloudFormation template enables the Changelog feature for S3 buckets.
+It deploys an SQS queue and an EventBridge rule which sends S3 update events to the queue.
+Then, the Elastio `iscan` job reads those events to perform the scanning of new objects.
+
+There are a couple of scenarios where this feature can be useful:
+
+* Incremental S3 scanning. For large buckets, scanning all objects might take a lot of time.
+    The changelog queue allows the scanning of only new objects in the bucket, significantly improving the
+    scan performance after the initial scan of the entire bucket is done.
+* Quasi-realtime scanning. Instead of running scan jobs according to a schedule in the protection
+    plan, Elastio can scan new objects in an S3 bucket continuously. As new S3 Object change events
+    arrive at the changelog queue, the `iscan` job is scheduled shortly after, allowing for
+    near-realtime scanning.
+
+## Deploying the CFN stack
+
+1. First, you need to enable Amazon EventBridge for your S3 buckets by following these instructions:
+    [Enabling Amazon EventBridge](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications-eventbridge.html).
+2. You can use 
+    [this quick-create link](https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/review?templateURL=https://elastio-artifacts-kskorokhodov-us-east-2.s3.us-east-2.amazonaws.com/cloudformation.yaml&stackName=elastio-s3-changelog)
```

Comment:
Could you please replace this link as well?

Reply:
Thanks for pointing to this. Updated